src/App.jsx

```diff
@@ -20,25 +21,30 @@ export const App = () => (
   <div className="section">
     <div className="container">
       <h1 className="title is-2">React Article</h1>
+      <>
+        <Article
+          title={firstArticle.title}
+          date={firstArticle.date}
+          text={firstArticle.text}
```

Comment:
pass whole object as a prop and destructure it inside component

Reply:
Did props in my component aren't destructured already?

Reply:
In Misha's lesson Prop destructuring thats what he's calling destructuring of props.
But I found the way.